CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v3.2.0](https://github.com/voxpupuli/puppet-epel/tree/v3.2.0) (2021-11-26)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-epel/compare/v3.1.0...v3.2.0)
+
+**Breaking changes:**
```

Comment:
Breaking changes should lead to a bump of the major version component if we follow semver.

Reply:
Amended to 4.0.0 - I wasn't originally sure if it deserved a full version bump as the breaking change was only to the declared support in the metadata and doesn't functionally break the API.